addons/mail/static/tests/tours/discuss_channel_call_as_guest_tour.js

```diff
@@ -0,0 +1,26 @@
+import { registry } from "@web/core/registry";
+
+registry.category("web_tour.tours").add("discuss_channel_call_as_guest_tour.js", {
+    test: true,
+    steps: () => [
+        {
+            content: "The call does not start on the welcome page",
+            trigger: ".o-mail-WelcomePage",
+            async run() {
+                await new Promise((r) => setTimeout(r, 250));
+                if (odoo.__WOWL_DEBUG__.root.env.services["discuss.rtc"]?.selfSession) {
+                    console.error("The call should not have started.");
+                }
```

Comment:
Also check rtc session the same way at the end of the test, otherwise negative assert is very weak.